scripts/smoketest.py

```diff
@@ -1,37 +1,48 @@
+import io
 import os
 import time
+import gzip
+import shutil
 from pathlib import Path
+from typing import List
+import tempfile
 
 import pytest
 import requests
 import pandas as pd
-import io
 
 API_URL = "http://localhost:8000/general/v0/general"
 # NOTE(rniko): Skip inference tests if we're running on an emulated architecture
 skip_inference_tests = os.getenv("SKIP_INFERENCE_TESTS", "").lower() in {"true", "yes", "y", "1"}
 
 
 def send_document(
-    filename,
-    content_type,
-    strategy="auto",
-    output_format="application/json",
-    pdf_infer_table_structure="false",
+    filenames: List[str],
+    filenames_gzipped: List[str],
```

Comment:
The looking weird empty lists simulate a case that can happen in practice, because users can pass a empty string as `uncompressed_content_type`, and the API should then should detect the filetype after uncompressing. So instead of changing logic, I'd consider either:
1. improving comment above (` compressed and uncompressed, guess the content-type` -> ` compressed and uncompressed \n The empty content-type means that API should detect filetype after uncompressing`), 
2. or passing a argument like `"detect-content-type"` which will be replaced by "", to make it more clear what's happening here: ` if uncompressed_content_type == "detect-content-type": uncompressed_content_type=""`. 

I vote for 1 :)

Reply:
Ah, didn't look in the files list until now, I thought you commented about it on the gzip test function and content-type, but it is actually about filenames_gzipped in this one. I totally agree that calls to send_document look weird now, and propose to just use named parameters everywhere. Will push fix in a minute.